public-ui/src/app/data-browser/views/survey-view/survey-view.component.html

```diff
@@ -96,8 +96,15 @@ <h2 class="secondary-display">
                   <clr-icon *ngIf="!showAnswer[q.conceptId]" shape="caret" dir="right"></clr-icon>
                   <clr-icon *ngIf="showAnswer[q.conceptId]" shape="caret" dir="down"></clr-icon>
                 </a>
+
+ 
+
               <ng-container *ngIf="q.expanded && q.resultFetchComplete">
                 <chart-survey-answers *ngIf="isCopeSurvey"[countAnalysis]="q.countAnalysis.results" [versionAnalysis]='q.versionAnalysis.results'></chart-survey-answers>
+                
+                <app-survey-answer-react [isCopeSurvey]="isCopeSurvey" [question]="q" [totalPartcipants]="survey.participantCount">
```

Comment:
The `totalPartcipants` input here is named `particpantCount` in the React component's props, just need to make sure they match